pyroll/core/profile/profile.py

```diff
@@ -17,6 +17,9 @@ class Profile(HookHost):
     cross_section = Hook[Polygon]()
     """Shape of the profile's cross-section."""
 
+    cross_section_area_deviation = Hook[float]()
```

Comment:
Definition is unclear. Suggest to introduce `RollPass.target_filling` and additonal `RollPass.OutProfile.filling_error` resp. `RollPass.OutProfile.cross_section_error` equivalent to existing filling ratios but based on target filling.